### ISSUE TITLE
CRAN submission

### ISSUE THREAD

**Issue by rafapereirabr** [ipeaGIT/r5r#155]
feedback from cRAN:

> Thanks, we see:
> 
> 
>    License components with restrictions and base license permitting such:
>      MIT + file LICENSE
>    File 'LICENSE':
>      MIT License
> 
>      Copyright (c) 2020 Ipea
> 
>      Permission is ...
> 
> Please only ship the CRAN template for the MIT license.
> 
> 
>    Size of tarball: 5307508 bytes
> 
> Not more than 5 ;B for a CRAN package, please.
> 
> 
> Please fix and resubmit.
> 
> Best,
> Uwe Ligges

I will fix the License asap. Can any of you guys have a look at the package size?

**Comment by mvpsaraiva** [comment]
I guess the accessibility map I've added to `inst/img` caused the package size issue. I've added that folder to .Rbuildignore, since they don't need do be installed locally because the vignettes read those images directly from GitHub's servers.

**Comment by rafapereirabr** [comment]
It seems it did not work. This is the warning I'm still getting:

```R
# CRAN
Sys.setenv(NOT_CRAN = "false")
devtools::check(pkg = ".",  cran = TRUE, env_vars = c(NOT_CRAN = "false"))
```

> checking installed package size ... NOTE
    installed size is  5.6Mb
    sub-directories of 1Mb or more:
      doc       1.4Mb
      extdata   3.3Mb

**Comment by rafapereirabr** [comment]
I've now added `^man/figures` to `.Rbuildignore`.

**Comment by rafapereirabr** [comment]
Another comment from CRAN:

> We still see the message
> 
>  > r5r_core <- setup_r5(data_path = data_path)
> Error in setup_r5(data_path = data_path) :
>    This package requires the Java SE Development Kit 11.0.8
>                          or higher. Please update your Java
> installation. The jdk
>                          11.0.8 can be freely downloaded from
> 
> https://www.oracle.com/java/technologies/javase-jdk11-downloads.html
> Execution halted
> 
> 
> Note that 1. a new Java is not available for all platforms and 2. your
> sentence above is only partly true, as the version you cite above can be
> freely downloaded but not freely be used with the new license from
> Oracle. S please point to a FOSS alternative.
> 
> Best,
> Uwe Ligges

**Comment by rafapereirabr** [comment]
The issue with Java version has been fixed in #157
